openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/custom-node.less

```diff
@@ -113,6 +123,23 @@
   top: 38px; // Need to show handles on top half
 }
 
+.react-flow .lineage-column-node-handle {
+  width: 20px;
+  border-radius: 0;
+  border-color: #0000001a;
+  background: #fff;
+  top: 0;
```

Comment:
Did we remove box-shadow from all the buttons on the Lineage

Reply:
Yes we have removed box-shadow